.rubocop.yml

```
@@ -8,3 +8,6 @@ AllCops:
 
 Layout/EmptyLineAfterGuardClause:
   Enabled: false
+
+Rails/RequestReferer:
+  EnforcedStyle: referrer
```

```suggestion
  EnforcedStyle: referrer
  
```
Uh oh! Needs a new line.

---

Got it, homie!